vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageInformationData.kt

```diff
@@ -58,11 +58,14 @@ data class ReferencesInfoData(
 data class ReactionsSummaryData(
         /*List of reactions (emoji,count,isSelected)*/
         val reactions: List<ReactionInfoData>? = null,
-        val showAll: Boolean = false,
+        val showAll: Boolean = false
+) : Parcelable
+
+data class ReactionsSummaryEvents(
         val onShowMoreClicked: () -> Unit,
         val onShowLessClicked: () -> Unit,
         val onAddMoreClicked: () -> Unit
-) : Parcelable
+)
```

Comment:
I though the idea was to avoid having lambda in data class. Anyway I understand why it should fix the crash.

On my side, I would have added a new sub interface to [TimelineEventController.Callback](https://github.com/vector-im/element-android/blob/main/vector/src/main/java/im/vector/app/features/home/room/detail/timeline/TimelineEventController.kt#L116), but I did not try to do so (this is maybe not better).

Reply:
I would be up for avoiding the bundle of functions by passing an interface instead (not a blocker for this PR from me)

Reply:
I Suspect the problem is due to the `@Parcelize` and not the data class. If the crash continue then we will change it with a callback.

I also tried with callback at first but then you would have to implement that callback on `TimelineFragment` and we don't want that there. I think this is a bit more clear with lamda